Makefile

```
@@ -1,16 +1,17 @@
 up:
 	docker-compose up -d
+down:
+	docker-compose down -v
```

The clean target uses `-v` because it also wants to clean out the volumes. A straight `down` shouldn't do that. So this should be:
```suggestion
	docker-compose down
```

---

thanks @mogul. This has been updated.